ticdc/ticdc-changefeed-config.md

```diff
@@ -111,16 +127,33 @@ dispatchers = [
 # The protocol configuration item specifies the protocol format of the messages sent to the downstream.
 # When the downstream is Kafka, the protocol can only be canal-json or avro.
 # When the downstream is a storage service, the protocol can only be canal-json or csv.
+# Note: ignore this field when the downstream is the Database.
 protocol = "canal-json"
 
 # The following three configuration items are only used when you replicate data to storage sinks and can be ignored when replicating data to MQ or MySQL sinks.
 # Row terminator, used for separating two data change events. The default value is an empty string, which means "\r\n" is used.
 terminator = ''
-# Date separator type used in the file directory. Value options are `none`, `year`, `month`, and `day`. `none` is the default value and means that the date is not separated. For more information, see <https://docs.pingcap.com/tidb/dev/ticdc-sink-to-cloud-storage#data-change-records>.
+# Date separator type used in the file directory. Value options are `none`, `year`, `month`, and `day`. `none` is the default value and means that the date is not separated. For more information, see <https://docs.pingcap.com/tidb/dev/ticdc-sink-to-cloud-storage#data-change-records>. Note: ignore this field when replicating data to the downstream other than Storage.
```

Comment:
```suggestion
# Date separator type used in the file directory. Value options are `none`, `year`, `month`, and `day`. `none` is the default value and means that the date is not separated. For more information, see <https://docs.pingcap.com/tidb/dev/ticdc-sink-to-cloud-storage#data-change-records>. Note: ignore this field when replicating data to the downstream other than a storage service.
```

Reply:
if it is not a parameter name, can we use a general way to mention it? 
"storage service" is usually used in CDC docs, such as [Replicate Data to Storage Services | PingCAP Docs](https://docs.pingcap.com/tidb/stable/ticdc-sink-to-cloud-storage).